ddtrace/gateway/__init__.py

```diff
@@ -17,4 +17,4 @@ class ADDRESSES(Enum):
     # SERVER_RESPONSE_TRAILERS = "server.response.trailers"
 
 
-__all__ = ["ADDRESSES", "Gateway"]
+__all__ = ["_Addresses", "_Gateway"]
```

Comment:
Is this submodule supposed to be public or private? Is the `*`-export of what look like private classes deliberate?

Reply:
I removed the exports. For now, it should be all private - good catch!